frontend/index.js

```diff
@@ -379,11 +379,15 @@ async function handleHashChange() {
     history.replaceState({}, "", "#welcome")
   }
   const { source, crumbs } = await fetchSourceWithCrumbs(opts)
+
+  editor.onUpdate(null)
   editor.update({ value: source, errorLines: {} })
+
   document.querySelector(".editor-wrap").scrollTo(0, 0)
   crumbs && updateBreadcrumbs(crumbs)
   clearOutput()
   await format()
+  editor.onUpdate(clearHash)
```

Comment:
I think you should also call `clearHash` here if `opt.content` is populated. The content urls are ugly and not really useful to leave in the URL bar once the content is loaded. It's annoying to have to edit it off (because its so long) when you frob with URLs. So I favor clearing it once the content has been loaded from it.

Reply:
Ok, I could got with that. If and when URLs become shorter snippet hashes that can be re-evaluated. For examples I think the URL should stay: it's meaningful and easily copyable.

Reply:
Done.